modules/client-common/src/main/java/org/apache/ignite/internal/jdbc/proto/event/JdbcColumnMeta.java

```diff
@@ -390,6 +392,10 @@ private static String typeName(String cls) {
             return "BINARY";
         } else if (Time.class.getName().equals(cls)) {
             return "TIME";
+        } else if (Duration.class.getName().equals(cls)) {
```

Comment:
could you please provide a test to validate that metadata of all types are properly propagated?

Reply:
ItJdbcMetadataSelfTest#testColumnTypesDescribed